### DISCUSSION TITLE
Error when running npm install

### DISCUSSION THREAD

**Discussion by cmw9706** [floating-ui/floating-ui#2367] [Q&A]
Hello,

I would love to contribute to this library but I am getting the following error whenever I run ```npm install``` 

```
 npm install

> @floating-ui/monorepo@0.0.0 postinstall /Users/connor.williams/Projects/floating-ui
> patch-package

patch-package 6.5.0
Applying patches...
Error: Patch file found for package runtime-dom which is not present at node_modules/@vue/runtime-dom
```

It looks like its trying to patch a @vue package in my node_modules, however ```@vue``` is not a dependency defined in package.json.

Has anyone experiences this before? I have tried to clean and reinstall with no luck. I am just following the [Contribute](https://github.com/floating-ui/floating-ui#contributing) section of the repo. Thanks!

**Comment by atomiks** [comment]
It looks like we can get rid of the patch now: https://github.com/vuejs/core/pull/7958 cc: @lozinsky 

Edit: Ah I don't think so yet. Only in v3.4 which isn't yet released.

OT: I'm not sure why you're getting the message — I can do a clean install fine. Have you got Vue installed globally? What's your OS?

**Comment by lozinsky** [comment]
I tried a clean install and couldn't reproduce it either. After you got the error, does `node_modules` contain the `vue` (not `@vue/*`) package? If so, what version is it?

**Comment by cmw9706** [comment]
Update @lozinsky @atomiks :

I updated my node version using nvm to ```v16.18.0``` and ran install again and got the follow error 

```
While resolving: @floating-ui/react-native@0.1.0
 Found: react@18.2.0
 node_modules/react
   dev react@"^18.0.0" from @floating-ui/react-native@0.1.0
   packages/react-native
     @floating-ui/react-native@0.1.0
     node_modules/@floating-ui/react-native
       workspace packages/react-native from the root project
 
 Could not resolve dependency:
 peer react@"17.0.2" from react-native@0.68.7
 node_modules/react-native
   dev react-native@"^0.68.0" from @floating-ui/react-native@0.1.0
   packages/react-native
     @floating-ui/react-native@0.1.0
     node_modules/@floating-ui/react-native
       workspace packages/react-native from the root project
```
so then i tried running

```
npm install --force
```

Got the following message!
![image](https://github.com/floating-ui/floating-ui/assets/11686150/b9579a2a-b74c-453b-981c-986cbdce949f)

But it looks like things are building now and hopefully this problem is behind me

 @atomiks I have Vue installed globally and am running on macOS Monterey Version 12.4
 
 @lozinsky After I ran the force install, node_modules contains BOTH ```@vue```  and ```vue```
